### DISCUSSION TITLE
Clarification for parameters in NoahmpTable.TBL

### DISCUSSION THREAD

**Discussion by psbecker** [NCAR/noahmp#155] [General]
I am looking at the parameters in the Noah parameter table and noticed that for the soil types, there are the "noahmp_stas_soil_categories" and "noahmp_soil_stas_ruc_parameters". 
The parameters for both are the same but the values are different. I'm not sure which gets used in the model though as it's not like the vegetation type where we indicate if it's MODIS or USGS.

Additionally, for the Option 3 runoff model, it says that permanent wilting point soil moisutre, saturated soil moisture, ground frozen factor, and soil moisture field capacity can all be found in the NoahmpTable.TBL. However, I'm not exactly sure which parameters match it as the names don't match. Could someone confirm that the parameters are what is listed in the table as below?

- Saturated soil moisture (theta_soil,max)=MAXSMC
- permanent wilting point soil moisture (theta_soil,wilt)=WLTSMC
- frozen soil factor (C_frz)=REFKDT_DATA
- Ground frozen factor (F_frz,g)=FRZK_DATA
- soil moisture field capacity (theta_soil,ref)=REFSMC
- soil hydraulic conductivity (K_soil,cond) = SATDK or is it something else?

Thanks in advance

**Comment by tslin2** [comment]
In the NoahmpTable.TBL, the below part determine,


&noahmp_stas_soil_categories
 SLTYPE  = "STAS"  ! soil dataset: "STAS" or "STAS_RUC"
 SLCATS  = 19      ! num_soil_types
/

"STATS" use noahmp_soil_stas_parameters
"STAS_RUC" use noahmp_soil_stas_ruc_parameters

**Comment by tslin2** [comment]
SATDK is saturated soil hydraulic conductivity

**Reply by tslin2** [reply]
REFKDT is Reference Soil Infiltration Parameter (used in runoff formulation)

**Comment by cenlinhe** [comment]
MAXSMC is the Saturated soil moisture
WLTSMC is the wilting point soil moisture
REFSMC is the reference soil moisture (field capacity)
For REFKDT and FRZK, they are used to compute soil impervious fraction coefficient: https://github.com/NCAR/noahmp/blob/fa2d2098bac1e437a21ccc15494b152996d5bc84/drivers/hrldas/WaterVarInTransferMod.F90#L214-L216
The physical meaning of each soil properties is also provided in the NoahMP parameter table comment part: https://github.com/NCAR/noahmp/blob/master/parameters/NoahmpTable.TBL#L764-L778
